src/entities/email_campaign.entity.ts

```diff
@@ -0,0 +1,22 @@
+import { Column, Entity, PrimaryGeneratedColumn } from 'typeorm';
+
+import { CAMPAIGN_TYPE } from 'src/utils/constants';
+import { BaseEntity } from './base.entity';
+
+@Entity({ name: 'email_campaign' })
+export class EmailCampaignEntity extends BaseEntity {
+  @PrimaryGeneratedColumn('uuid')
+  id: string;
+
+  @Column()
+  campaignType: CAMPAIGN_TYPE;
+
+  @Column({ unique: true })
```

Comment:
@swetha-charles  are you sure you want this to be unique?

Reply:
For now, I think yes. Booking code which will be the campaign info needs to be unique. If needed, I am thinking we change it to be non unique

Reply:
👍  okay good to know. I wasn't sure what campaignInfo was. It sounds a little like a description. Now I know its a booking code it makes more sense